weblate_musicbrainz_format_check/checks.py

```diff
@@ -65,12 +65,12 @@ def check_single(self, source, target, unit):
 
         source_matches = MUSICBRAINZ_BRACE_MATCH.findall(source)
         source_identifiers = sorted(set(
```

Comment:
@yvanzo Not sure about sorted + set. If we want to compare as a set, then we don't need the sorted. But it depends on what is expected. E.g.

```
is/was {additional:an|a} {additional} {original} {eponymous} member of
```

Is it important to have exactly the same number of occurences, that is two times `additional` and one time the other placeholders? Then we should leave the `sorted` but remove the `set`

Reply:
I just didn’t play with `set` and `sorted` for a long time.
What is expected is described above in comment to the function.

Reply:
So it can be used any additional number of times. Also less?

Reply:
https://github.com/metabrainz/musicbrainz-weblate-checks/pull/1/commits/58fbdb7570ef08f3939f0470c1c954e686920260